packages/elements/src/components/app/app.component.ts

```diff
@@ -415,6 +421,13 @@ export class MutationTestReportAppComponent extends RealTimeElement {
       return nothing;
     }
   }
+
+  #openFilePicker() {
+    const picker = this.shadowRoot?.querySelector('mte-file-picker');
```

Comment:
I think it would be cleaner to either use a [ref](https://lit.dev/docs/api/directives/#ref), or a state attribute (`this.filePickerMode = 'open'`)